library/src/main/java/com/leinardi/android/speeddial/SpeedDialView.java

```diff
@@ -773,9 +811,14 @@ private void visibilitySetup(boolean visible, boolean animate) {
             }
         } else {
             for (int i = 0; i < size; i++) {
-                FabWithLabelView fabWithLabelView = mFabWithLabelViews.get(i);
+                int index = reverseAnimation ? size - 1 - i : i;
+                FabWithLabelView fabWithLabelView = mFabWithLabelViews.get(index);
                 if (animate) {
-                    UiUtils.shrinkAnim(fabWithLabelView, false);
+                    if (reverseAnimation) {
+                        hideWithAnimationFabWithLabelView(fabWithLabelView, i * 50);
```

Comment:
This was actually a mistake on my behalf (magic number), but since now is used twice (line 809 and here) please move both to a constant, perhaps called `ACTION_ANIM_DELAY`

- [ ] move to constant